codegen-renderers/src/main/kotlin/io/vrap/codegen/languages/java/file/producers/JavaFileProducer.kt

```diff
@@ -0,0 +1,40 @@
+package io.vrap.codegen.languages.java.file.producers
+
+import io.vrap.rmf.codegen.io.TemplateFile
+import io.vrap.rmf.codegen.rendring.FileProducer
+import io.vrap.rmf.codegen.rendring.utils.escapeAll
+import io.vrap.rmf.codegen.rendring.utils.keepIndentation
+
+class JavaFileProducer : FileProducer {
+
+    override fun produceFiles(): List<TemplateFile> {
+        return listOf(produceSphereRequestInterface())
+    }
+
+    private fun produceSphereRequestInterface() : TemplateFile {
+        val content = """
+            |package com.commercetools.importer.commands;
```

Comment:
This really raises an interesting question: Should we generate all required static code or should we put this code to a library?